README.md

```diff
@@ -36,17 +36,25 @@ You can follow these instructions to build an ngraph Python wheel containing bot
 
 https://github.com/NervanaSystems/ngraph/blob/master/python/README.md
 
+nGraph build process on Ubuntu 16.04:
+
+    # apt update
+    # apt install python3 python3-pip python3-dev
+    # apt install build-essential cmake curl clang-3.9 git zlib1g zlib1g-dev libtinfo-dev
+    $ git clone https://github.com/NervanaSystems/ngraph.git
+    $ cd ngraph/python
+    $ ./build_python3_wheel.sh
```

Comment:
To prevent from building once again nGraph I would recommend to pass `-DNGRAPH_INSTALL_PREFIX=/path/to/installed/ngraph_dist ..`. Thus one should execute following commands:

```
rm -rf build
mkdir build
cd build && cmake -DNGRAPH_INSTALL_PREFIX=/path/to/installed/ngraph_dist .. && make -j8 && python3 setup.py bdist_wheel
```

Reply:
And use CMAKE instead of this script